src/extensions/FileUploadExt.ts

```
@@ -0,0 +1,2205 @@
+  
+  // Apply editable enhancements to each image
+  images.forEach(img => {
+    // Check if image is already enhanced
+    if (img.getAttribute('data-enhanced') === 'true') {
+      return; // Skip already enhanced images
+    }
+    
+    // Force all images to be editable in PDFs
+    const parent = img.parentNode as HTMLElement;
+    if (parent && !(parent.classList && parent.classList.contains('editable-image-container'))) {
+      // Replace the image with our editable version
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)